scholly/base/src/global/fakeData.js

```diff
@@ -0,0 +1,31 @@
+export const FAKE_PUBLICATIONS = [
+  {
+    description: 'First Publication',
+    category: {
+      type: 'Journal',
+      categoryTitle: 'Example Journal',
+      issue: '1.1',
+      volume: '0.2',
+      pages: '20',
+      publisher: 'Penguine books',
+    },
+    link: 'thislinkisfakedonteventrytoclickit.okay',
+    authors: ['KK', 'LOL'],
+    yearPublished: '2007',
+  },
```

Comment:
Please add > 1 publication to see the effects of having more publications that can fit in one page, hence needing scrolling.

Reply:
Yep I was just a bit too lazy to make more fake data 😂